plugins/woocommerce-admin/client/products/fields/attribute-input-field/attribute-input-field.tsx

```diff
@@ -65,51 +88,80 @@ export const AttributeInputField: React.FC< AttributeInputFieldProps > = ( {
 		: null;
 
 	return (
-		<SelectControl< Pick< QueryProductAttribute, 'id' | 'name' > >
```

Comment:
This is just indented as I had to add a Fragment around this in order to add the create modal